bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/internal/AudioConsoleCommandExtension.java

```diff
@@ -75,6 +80,8 @@ public void execute(String[] args, Console console) {
                 case SUBCMD_SINKS:
                     listSinks(console);
                     return;
+                case SUBCMD_GROUPS:
+                    handleGroups(args, console);
```

Comment:
This case should contain a break / return / throw at the end regardless if the default case contains a logic or not.